iterator/cdc_iterator.go

```diff
@@ -0,0 +1,212 @@
+package iterator
+
+import (
+	"context"
+	"fmt"
+	"github.com/conduitio-labs/conduit-connector-airtable/config"
+	"github.com/conduitio-labs/conduit-connector-airtable/position"
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	airtableclient "github.com/mehanizm/airtable"
+	"time"
+)
+
+const LASTMODIFIED = "last-modified"
+const LASTMODIFIEDSTR = "last-modified-str"
```

Comment:
It's great that you extracted this into constants! A nitpick related to this: it's a convention to use mixed caps for naming constants (in Go at least): https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps.